source/common.misc.datasheet.bmx

```diff
@@ -56,12 +56,12 @@ Type TDatasheetSkin
 		skin.textColorBad = new SColor8(80,10,10)
 		skin.defaultFont = GetBitmapFontManager().baseFont
 
-		skin.fontNormal = GetBitmapFontManager().baseFont
-		skin.fontBold = GetBitmapFontManager().baseFontBold
+		skin.fontNormal = GetBitmapFontManager().Get("default", 11)
+		skin.fontBold = GetBitmapFontManager().Get("default", 12, BOLDFONT)
 		skin.fontSmall = GetBitmapFontManager().Get("default", skin.fontNormal.FSize-1)
 		skin.fontSemiBold = GetBitmapFontManager().Get("defaultThin", -1, BOLDFONT)
-		skin.fontCaption = GetBitmapFontManager().Get("default", 13, BOLDFONT)
-		skin.fontSmallCaption = GetBitmapFontManager().Get("default", 12, BOLDFONT)
+		skin.fontCaption = GetBitmapFontManager().Get("default", 12, BOLDFONT)
```

Comment:
Ich glaube, an diesem Punkt tut mir die neue Schriftart am meisten weh. 800x600 OK, Vollbild OK, in allen anderen Fenstergrößen, die ich bei mir probiert habe (für Entwicklung standardmäßig 1024x768), sehen die Titel in den Datenblättern furchtbar aus.
Wenn ich die Schriftgröße hier auf 11 setze, sieht es viel besser aus.

Vielleicht "skaliert" diese Schriftart nicht so reibungslos wie die alte.

Reply:
Das Problem "Skalierung" muessen wir fuer diesen PR ignorieren..bzw darauf keine Ruecksicht nehmen.

Derzeit wird jede Glyphe einzeln skaliert ..und das kann und wird doof aussehen.
Manche Elemente nutzen schon TextBox-caches. Dann wird bspweise der gesamte Titel als ein Bild gecached und somit als ein ganzes Element skaliert. Davon profitieren vorallem Zeichenabstaende, Antialiasing untereinander usw.

Spaeter wird aber wohl alles in nativer Aufloesung in einer einzelnen Textur landen, und dann erst skaliert werden (smooth oder pixelig...).